lightning/src/ln/msgs.rs

```diff
@@ -1698,6 +1698,13 @@ mod fuzzy_internal_msgs {
 			amt_to_forward: u64,
 			outgoing_cltv_value: u32,
 		},
+		#[allow(unused)]
+		TrampolineEntry {
+			amt_to_forward: u64,
+			outgoing_cltv_value: u32,
+			multipath_trampoline_data: Option<FinalOnionHopData>,
+			trampoline_packet: crate::onion_message::packet::Packet,
```

Comment:
Did we figure out that this struct is reusable in the trampoline case? If so we should probably move it to a different module to avoid the `onion_message` module prefix.